.github/workflows/build.yml

```
@@ -0,0 +1,108 @@
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+      - name: 빌드 결과 설정
+        if: always()
+        id: build-result
+        run: |
+          if [ "${{ steps.build.outcome }}" = "success" ]; then
+            echo "status=success" >> $GITHUB_OUTPUT
+          else
+            echo "status=failure" >> $GITHUB_OUTPUT
+          fi
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)